BUILD.gn

```diff
@@ -66,69 +58,82 @@ group("dart_sdk") {
 group("flutter") {
   testonly = true
 
-  # Compile the engine.
-  public_deps = [
-    ":unittests",
-    "//flutter/shell/platform/embedder:flutter_engine",
-    "//flutter/sky",
-  ]
-
-  # Ensure the example for a sample embedder compiles.
-  if (build_embedder_examples) {
-    public_deps += [
-      "//flutter/examples/glfw",
-      "//flutter/examples/vulkan_glfw",
+  # If building for WASM, only build CanvasKit.
+  if (is_wasm) {
+    deps = [ "//third_party/skia/modules/canvaskit:canvaskit" ]
+  } else {
```

Comment:
Is there any reason to call this target `flutter` anymore?

Why not just have a top level `canvaskit` target that asserts `is_wasm` and depends on the canvaskit?

We could then leave the flutter group alone. WDYT?

Reply:
Done.